src/Uno.UI/UI/Xaml/Controls/MediaPlayer/MediaTransportControls.cs

```diff
@@ -0,0 +1,301 @@
+	public partial class MediaTransportControls : Control
+	{
+		private const string RootGridName = "RootGrid";
+		private const string PlayPauseButtonName = "PlayPauseButton";
+		private const string PlayPauseButtonOnLeftName = "PlayPauseButtonOnLeft";
+		private const string VolumeMuteButtonName = "VolumeMuteButton";
+		private const string AudioMuteButtonName = "AudioMuteButton";
+		private const string VolumeSliderName = "VolumeSlider";
+		private const string FullWindowButtonName = "FullWindowButton";
+		private const string CastButtonName = "CastButton";
+		private const string ZoomButtonName = "ZoomButton";
+		private const string PlaybackRateButtonName = "PlaybackRateButton";
+		private const string SkipForwardButtonName = "SkipForwardButton";
+		private const string NextTrackButtonName = "NextTrackButton";
+		private const string FastForwardButtonName = "FastForwardButton";
+		private const string RewindButtonName = "RewindButton";
+		private const string PreviousTrackButtonName = "PreviousTrackButton";
+		private const string SkipBackwardButtonName = "SkipBackwardButton";
+		private const string StopButtonName = "StopButton";
+		private const string AudioTracksSelectionButtonName = "AudioTracksSelectionButton";
+		private const string CCSelectionButtonName = "CCSelectionButton";
+		private const string TimeElapsedElementName = "TimeElapsedElement";
+		private const string TimeRemainingElementName = "TimeRemainingElement";
+		private const string ProgressSliderName = "ProgressSlider";
+		private const string BufferingProgressBarName = "BufferingProgressBar";
+		private const string TimelineContainerName = "MediaTransportControls_Timeline_Border";
+		private const string HorizontalThumbName = "HorizontalThumb";
+		private const string DownloadProgressIndicatorName = "DownloadProgressIndicator";
+
+		private Grid _rootGrid;
+		private Button _playPauseButton;
+		private Button _playPauseButtonOnLeft;
+		private Button _volumeMuteButton;
+		private Button _audioMuteButton;
+		private Slider _volumeSlider;
+		private Button _fullWindowButton;
+		private Button _castButton;
+		private Button _zoomButton;
+		private Button _playbackRateButton;
+		private Button _skipForwardButton;
+		private Button _nextTrackButton;
+		private Button _fastForwardButton;
+		private Button _rewindButton;
+		private Button _previousTrackButton;
+		private Button _skipBackwardButton;
+		private Button _stopButton;
+		private Button _audioTracksSelectionButton;
+		private Button _ccSelectionButton;
+		private TextBlock _timeElapsedElement;
+		private TextBlock _timeRemainingElement;
+		private Slider _progressSlider;
+		private ProgressBar _bufferingProgressBar;
+		private Border _timelineContainer;
+		private ProgressBar _downloadProgressIndicator;
+
+		private Timer _controlsVisibilityTimer;
+		private bool _wasPlaying;
+
+		public MediaTransportControls() : base()
+		{
+			_controlsVisibilityTimer = new Timer()
+			{
+				AutoReset = false,
+				Interval = 3000
```

Comment:
?

Reply:
Totally arbitrary value, as there is no way to setup the delay during which the MTC is displayed in original video player